carla_twist_to_control/src/carla_twist_to_control/carla_twist_to_control.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# This work is licensed under the terms of the MIT license.
```

Comment:
If we don't already, let's make sure we have a `LICENSE` file with the contents of the MIT license in the directory for this package, and probably call out that this 'carla_twist_to_control` package is MIT licensed somewhere in the readme. Might not hurt to check on the licenses of whatever other packages we have here.

Reply:
We are not using this anymore. It was here to replace the ackermann control due to the performance of ackermann control. Do we want to keep it or delete it? If we want to keep it, I will check the license.

Reply:
Deleted it